message/tests.py

```diff
@@ -45,7 +45,7 @@ def saved_court(self):
     def saved_game_event(self, saved_court):
         return GameEvent.objects.create(id=TEST_ID, time=TEST_TIME, level_of_game=TEST_LEVEL,
                                         min_number_of_players=TEST_MIN, max_number_of_players=TEST_MAX,
-                                        court_id=saved_court, ball_game=TEST_BALL_GAME)
+                                        court=saved_court, ball_game=TEST_BALL_GAME)
```

Comment:
> for what was that change? there were any errors?

No, I'm just changing the field name to be court from court_id. Because your model
(Message) depending on game_event model, so after the changing on game_event,
I needed to change the name on your model too.

Reply:
@IdanZimi please indicate whether you're ok with closing this thread.

Reply:
@yftacherzog @MaayanMashhadi I understand why the change is necessary.